### DISCUSSION TITLE
Getting the time, ID, etc. of tiles.

### DISCUSSION THREAD

**Discussion by ATL-Flaneur** [gjoseph92/stackstac#166] [Q&A] [answered]
Doing `items = catalog.search(…).get_all_items()` gives an iterable collection of tiles that's traversable with `for item in items:`

How can I display the time, id, band, projection, or other metadata in the coordinates and attribute fields?

```
> item
<xarray.DataArray 'stackstac-17141b516efd194e112f6d943ea1c3f7' (y: 1944, x: 4212)>
array([[28.63329697, 28.29682732, 28.30708313, ..., 26.88253403,
        27.52575111, 28.17367172],
       [28.42001534, 28.24291611, 28.1334362 , ..., 26.72727585,
        26.65990639, 27.01013756],
       [28.40831757, 28.21773911, 27.94297028, ..., 26.78857231,
        26.57504272, 26.60680962],
       ...,
       [        nan,         nan,         nan, ...,         nan,
                nan,         nan],
       [        nan,         nan,         nan, ...,         nan,
                nan,         nan],
       [        nan,         nan,         nan, ...,         nan,
                nan,         nan]])
Coordinates:
    time             datetime64[ns] 2017-03-29
    id               <U10 'n41w075-13'
    band             <U4 'data'
  * x                (x) float64 -74.64 -74.64 -74.64 ... -74.25 -74.25 -74.25
  * y                (y) float64 40.12 40.12 40.12 40.12 ... 39.94 39.94 39.94
    proj:epsg        int64 5498
    gsd              int64 10
    threedep:region  <U6 'n40w80'
    epsg             int64 4326
Attributes:
    spec:        RasterSpec(epsg=4326, bounds=(-74.64, 39.94, -74.25, 40.12),...
    crs:         epsg:4326
    transform:   | 0.00, 0.00,-74.64|\n| 0.00,-0.00, 40.12|\n| 0.00, 0.00, 1.00|
    resolution:  9.259259259259259e-05
```

**Comment by gjoseph92** [comment, marked as answer]
@ATL-Flaneur I'm not quite sure what you're asking here. Time, ID, band, projection, and all other STAC metadata are stored in the coordinates of the DataArray (`item` in your case). If you're not familiar with the xarray data model, you might want to read https://docs.xarray.dev/en/stable/user-guide/data-structures.html#coordinates first to understand how coordinates work.

In this case, it looks like you've used stackstac on a single item. But `item.time`, `item.id`, `item.band`, `item["proj:epsg"]`, etc. will give you that metadata.

**Comment by ATL-Flaneur** [comment]
@gjoseph92 Thanks for the reply. Appending `.values` to the calls (e.g., `item.id.values`) returned the scalar value I was looking for.
